src/store/module/task.module.js

```diff
@@ -0,0 +1,40 @@
+const task = {
+  nameSpaced: true,
```

Comment:
We are not providing nameSpaced in camel case, Instead you can provide it as a single word **namespaced**